ensime_shared/util.py

```diff
@@ -0,0 +1,35 @@
+import os
+from contextlib import contextmanager
+
+class Util:
```

Comment:
I frankly think that we should remove the class and only import functions. After all, that's what we want and we want to import only the necessary utils.